csharp/tools/tracing-config.lua

```diff
@@ -21,6 +21,8 @@ function RegisterExtractorPack(id)
         -- if that's `build`, we append `-p:UseSharedCompilation=false` to the command line,
         -- otherwise we do nothing.
         local match = false
+        local matchTest = false;
```

Comment:
The variable here is named `matchTest`, but subsequently `testMatch` is used. Is that intentional or a typo?

Reply:
AUCH! I will fix this!